tests-web/www/js/import.js

```diff
@@ -27,13 +28,14 @@ export const makeImportCurrentView = (map, btn) => {
 
       btn.innerText = "Importing OSM data...";
 
-      const output = importOsm(osmXML, {
+      const network = new JsStreetNetwork(osmXML, {
         // TODO Ask overpass
         driving_side: "Right",
       });
 
       // TODO Definitely time to think about cleaning up old layers
-      L.geoJSON(JSON.parse(output), { style: { color: "#f55" } }).addTo(map);
+      makePlainGeoJsonLayer(network.toGeojsonPlain()).addTo(map);
+      makeDetailedGeoJsonLayer(network.toGeojsonDetailed()).addTo(map);
```

Comment:
We should consider providing mutually exclusive layers that can be used in conjunction, I think. Seems like a useful API going forward, as the different kinds of geom will be used in different ways. We can always combine the layers using a leaflet `LayerGroup` as needed for ease of use within the web app.

I was imagining "road area", "road markings" and "debug" (for whatever ad-hoc geom we want to render in garish colours during dev) as three initial outputs.

Reply:
Yeah, that makes more sense than having two mutually exclusive "renderings", one of which has a bunch of labeled layers anyway. I'll keep it this way for this PR and we can iterate further.

One of the things stopping us from lifting more of A/B Street's lane markings is the lack of turning/movement info. We can discuss that too shortly...